content/_events/2020/april.md

```diff
@@ -0,0 +1,126 @@
+---
+published: false
+upcoming: true
+title: Documentation templates for fun and profit
+event-id: april-2020-documentation-templates
+year: 2020
+event-date: Thursday, April 23, 2020
+locale: san-francisco
+location: google-345-spear-street
+rsvp_open: true
+description: |
+Find out how templates make better documentation.
+form:
+  id: id (don't include the #, for the modal dialog)
+  title: title for the form's modal dialog
+  embed: |
+   embed code for the form
+agenda: |
+    * 6:30 p.m. Socialize and network
+    * 7 p.m. Presentation, followed by questions
+    * 7:30 p.m. to 8:15 p.m. Work in groups
+    * 8:15 to 8:45 p.m. Socialize and pack up
+    * 9 p.m. Lights out
+
+## Documentation templates for fun and profit
+
+
+# Documentation Templates for Fun and Profit
+
+Erin McKean will speak about templates for documentation and her activities with The Good Docs Project, a group that aims to create templates for building or improving all sorts of documentation.
+
+Erin will cover the following:
+
+* Why do templates make better documentation?
+* Where can templates for documentation be found?
+* What should you do if you can't find a template?
+
+In a working session after the talk, visit one or more of these tables:
+
+* Get started contributing to The Good Docs Project.
+* Get help from The Good Docs Project to start or tune up a documentation project.
+* Join a round table discussion of documentation issues. Bring your questions and get feedback from other attendees. (And answer questions when you can.)
+
+OUR SPEAKER:
+Erin McKean is the Docs Advocacy Program Manager in the Open Source Programs Office at Google. You may also know her as the founder of the not-for-profit Wordnik.com, the world’s biggest online dictionary. Before founding Wordnik, she was the editor-in-chief of American Dictionaries for Oxford University Press, and before Javascript, she dabbled in HyperCard, Perl, and Omnimark (if you have ever written anything in Omnimark, she will buy you a drink). In addition to writing code, blog posts, and documentation, she’s also the author of the Weird and Wonderful Words books, the best-selling novel The Secret Lives of Dresses, and The Hundred Dresses, a field guide to dresses, and is an honorary fellow of the Society for Technical Communication. She blogs at dressaday.com. When you meet her, please tell her your favorite word.
+
+PROGRAM:
```

Comment:
So this is already in the "agenda" front matter variable, which displays it on the page. No need to have it twice.